takeshape/takeshape-starter-ecommerce-kitchen-sink
# Add loyalty points from voucherify to the pattern

### Test Plan (Steps to test):

1. Check preview site
2. Log in
3. inspect a `graphql` response in the network requests, look for the `loyaltyCard` data included in the profile response
4. Testing the webhook in preview would be pretty time consuming. Please just look over the code.

### Checklist:
